### PR TITLE
Handle `Global` with Babel

### PR DESCRIPTION
**Pull request by Andarist**
closes https://github.com/emotion-js/emotion/issues/1474

### CODE REVIEW

File: packages/babel-plugin-emotion/__tests__/import-mapping/__snapshots__/import-mapping.js.snap [hunk 1]
```diff
@@ -9,7 +9,7 @@ const SomeComponent = props => <div css={{ color: window }} {...props} />
 
       ↓ ↓ ↓ ↓ ↓ ↓
 
-import { someCssFromCore as _css } from \\"package-two\\";
+import { css as _css } from \\"@emotion/core\\";
```
Comment (Andarist) [inline packages/babel-plugin-emotion/__tests__/import-mapping/__snapshots__/import-mapping.js.snap]: seems like this broke - need to fix it

File: packages/babel-plugin-emotion/src/core-macro.js [hunk 2]
```diff
@@ -29,7 +34,40 @@ export const transformCssCallExpression = ({
...
+  )
+
+  transformCssCallExpression({
+    babel,
+    state,
+    path: expressionPath,
+    sourceMap
+  })
+
+  const { importSource = '@emotion/core', cssExport = 'css' } = state
```
Comment (Andarist) [inline packages/babel-plugin-emotion/src/core-macro.js]: seems like those should be configurable for import mapping case, any proposals for the API?

Comment (mitchellhamilton) [reply packages/babel-plugin-emotion/src/core-macro.js]: Require that modules that export `Global` also export `css`, this is already done for `jsx`.

Comment (Andarist) [reply packages/babel-plugin-emotion/src/core-macro.js]: Done

**Comment by Andarist** [comment]
@mitchellhamilton I've made changes around importMap and consider this to be done, please review 😄

File: .changeset/moody-stingrays-sort.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'babel-plugin-emotion': minor
+'@emotion/core': minor
+---
+
+`Global` gets handled by the Babel plugin now - this gives inline css-less expressions source maps. The same transforms are being applied for `Global` imported from macro entry (`@emotion/core/macro`).`
```
Comment (mitchellhamilton) [inline .changeset/moody-stingrays-sort.md]: Could you split this changeset into two, one for the babel plugin and one for core?

Comment (Andarist) [reply .changeset/moody-stingrays-sort.md]: Done

**Pull request merged**